lib/vault/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.6'
```

Comment:
see `lib/templates/docker-compose.yml` and `lib/templates/Dockerfile` for examples